### PR TITLE
Added 2 fuzzers

### PR DESCRIPTION
**Pull request by AdamKorcz**
I am bringing the 2 fuzzers from https://github.com/dragonflyoss/Dragonfly/pull/865 back into the upstream repository.

As agreed with @lowzj we are applying to run the fuzzers on oss-fuzz. I have setup an integration application [here](https://github.com/google/oss-fuzz/pull/3790). To complete that application, we need the 2 fuzzers merged here on the upstream repo.

### CODE REVIEW

File: dfget/core/uploader/uploader_fuzz.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 1999-2017 Alibaba Group..
```
Comment (lowzj) [inline dfget/core/uploader/uploader_fuzz.go:2]: It should be `Copyright The Dragonfly Authors.`

**Comment by lowzj** [comment]
Could you merge yours commits into one and [sign off](https://github.com/dragonflyoss/Dragonfly/pull/1338/checks?check_run_id=666389494) it?

**Comment by lowzj** [comment]
@AdamKorcz could I close the #865 if this pull request get merged

**Comment by AdamKorcz** [comment]
@lowzj This PR does not solve or replace #865, it adds just the fuzzers from that PR.

**Comment by lowzj** [comment]
could you rebase the code and combine your 5 commits into one commit ?

**Comment by AdamKorcz** [comment]
@lowzj Let me know if it looks better now.

**Comment by lowzj** [comment]
merged via manually, closed.

**Pull request closed without merging**